0020-day-two/stand-ups/readme.md

```diff
@@ -0,0 +1,99 @@
+
+1. What did you work on yesterday?
+2. What are you working on today?
+3. Do you have any impediments/blockers?
+
+After that point, the stand-up is concluded. There may be a parking lot discussion after. However, participation in the parking lot discussion is optional for all members except those explicitly needed for discussion of the issues raised.
+
+The term parking lot refers to a bucket of comments, concerns, or questions that will be discussed and/or addressed at a later point with potentially fewer contributors. This is part of a strategy to avoid letting the discussion in a meeting shift to a subject that is not aligned with the meeting goals and/or decisions.
+
+## Goals
+
+1. Bring awareness to the contribution of each member for the past and upcoming day.
+2. Surface any impediments to one or more team members' contributions.
+
+## Participation
+
+The entire team should attend the stand-up. Anyone that worked on a task towards the sprint work should answer the three questions. It would be up to the team to decide if they would like updates from members that are not directly working against sprint task work (i.e. Product Owners and Program Managers).
+
+- Scrum Master (Required)
+- Product Owner (Optional)
+- Program Manager (Required)
+- Dev Manager (Required)
+- Tech Lead + Contributors (Required)
+
+## Impact
+
+Team members get a clear understanding of what development tasks are going on within the team which helps with collaboration. It also provides a time to address any challenges or blockers that may be stopping specific tasks from being completed, therefore helping with velocity.
+
+## Measures
+
+Both stand-up length and time to start are important as the stand-up has to be seen as a reliable and efficient meeting that facilitates communication of information versus unnecessary overhead.
+
+### Stand-up Length
+
+While the length can depend on the team size, if everyone is sticking to one line answers to the 3 key questions, it should be fairly easy to conclude within 5-10mins.
+
+- 1-5 min = Excellent
+- 5-10 min = Great
+- 10-15 min = Good
+- 15+ min = Needs Improvement
+
+### Time to Start
+
+How long after the scheduled start time did contributors begin providing updates?
+
+- 0-1 min = Excellent
+- 1-3 min = Great
+- 3-5 min = Good
+- 5+ min = Needs Improvement
+
+### New Tasks Created After Stand-up
+
+How many tasks are being generated after the stand-up that didn't exist before? This can indicate how much unplanned work is being done. If creating new tasks after stand-up becomes routine (especially for the same story), this could indicate the story is at higher risk of not being completed. It could also indicate a shift in focus to an unplanned objective.
+
+## Facilitation Guidance
+
+The scrum master should facilitate the stand-up meeting.
+
+### Answer 3 Simple Questions

```

Comment:
this section is incomplete and looks like a duplicate of an earlier section.

Reply:
Fixed in https://github.com/microsoft/code-with-engineering-playbook/pull/67/commits/bada48282c0264e80a373809d8ad902caf0d106e.